ui/packages/catalog-ui-search/src/main/webapp/react-component/location/gazetteer.js

```diff
@@ -60,7 +101,33 @@ class Gazetteer extends React.Component {
     }
     const { id } = suggestion
     const res = await this.fetch(`./internal/geofeature?id=${id}`)
-    return await res.json()
+    const data = await res.json()
+    let finalArea = getLargestBbox(
+      data.geometry.coordinates,
+      this.isMultiPolygon(data.geometry.coordinates)
+    )
+    let boundingbox = [
+      finalArea.minY,
+      finalArea.maxY,
+      finalArea.minX,
+      finalArea.maxX,
+    ]
+    return {
+      type: 'Feature',
+      geometry: {
+        type: 'Polygon',
+        coordinates: [
+          [
+            [boundingbox[2], boundingbox[0]],
```

Comment:
✏️ 	If we use the `finalArea` values directly, we can remove of the `boundingbox` intermediate